EnglishTranslation-master/UI.tsv

```diff
@@ -2245,7 +2245,7 @@ UI_20160718_002244	{@st59}Shows dead parts of defeated monsters.{/}	{@st59}몬
 UI_20160718_002245	{@st66b}Show Character Silhouettes{/}	{@st66b}캐릭터 실루엣 적용{/}	UI_20160425_002180	{@st66b} Apply Character Silhouettes	{@st66b} 캐릭터 실루엣 적용	
 UI_20160718_002246	{@st59}Characters hidden behind the background are shown in silhouettes.{/}	{@st59}캐릭터가 배경 뒤에 있을 때 실루엣으로 표시합니다.{/}				
 UI_20160718_002247	Do you want to use this %s?{/}	%s 을 사용하시겠습니까?{/}	UI_20151224_001958	Do you want to use this %s?{/}	%s 아이템을 사용하시겠습니까?{/}	
-UI_20160718_002248	Unable to Use :	사용제한 : %s				
+UI_20160718_002248	Usable until :	사용제한 : %s				
```

Comment:
I would suggest, `Expires In:` 

![image](https://user-images.githubusercontent.com/18273533/34666537-c3f47c18-f45c-11e7-861e-e8e0b23338fd.png)